stackstorm-tools/actions/yaml_reader.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+from st2actions.runners.pythonrunner import Action
```

Comment:
We already have `csv` and `xml` pack so maybe this action should go into `yaml` pack :)